usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -107,6 +107,9 @@ def construct_contract_response(requested_award_dict):
     response["recipient"] = create_recipient_object(transaction)
     response["place_of_performance"] = create_place_of_performance_object(transaction)
 
+    parent_award = fetch_parent_award_from_piid(award["parent_award_piid"])
```

Comment:
Should this also be added to the `contruct_idv_response` method below?

Reply:
IDVs already have their own method:
```
parent_award = fetch_parent_award_details(award["generated_unique_award_id"])
```
We can't use the same method because the IDV method uses the generated unique id of the award itself to check for parent awards, which doesn't work for non-idv contracts

Reply:
Woops. Definitely read right over that.